core/tests/puppeteer-acceptance-tests/utilities/common/puppeteer-utils.ts

```diff
@@ -410,6 +412,46 @@ export class BaseUser {
   getCurrentUrlWithoutParameters(): string {
     return this.page.url().split('?')[0];
   }
+
+  /**
+   * This function checks the exploration accessibility with help of explorationID.
```

Comment:
I think it's better to say "... checks the exploration accessibility by navigating to the exploration page based on the explorationID"

Reply:
done